actions/mysql_show_create_table_action_test.go

```diff
@@ -50,7 +50,7 @@ func TestMySQLShowCreateTable(t *testing.T) {
 
 		var expected string
 		switch {
-		case mySQLVersion == "8.0" && mySQLVendor == tests.OracleMySQL:
+		case mySQLVersion == "8.0" && (mySQLVendor == tests.OracleMySQL || mySQLVendor == tests.PerconaMySQL):
```

Comment:
There are no other vendors for 8.0. The whole second condition should be removed.

Reply:
Fixed.
Also updated the way we check the vendor because not all vendors and versions return the same string capitalization